### ISSUE TITLE
[Chore] Next release?

### ISSUE THREAD

**Issue by noraj** [SecureAuthCorp/impacket#799]
Python 3.8 introduced a warning.

It's already fixed:

https://github.com/SecureAuthCorp/impacket/blob/0056b61ca52e13acf05d101518976980f1592d85/examples/GetUserSPNs.py#L445

Last release was 09/2019, is a release planned soon ?

### Configuration 

impacket version:  v0.9.20
Python version:  3.8
Target OS:  ArchLinux

### Debug Output With Command String  

```
$ GetUserSPNs.py --help
/usr/bin/GetUserSPNs.py:438: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if userDomain is '':
Impacket v0.9.20 - Copyright 2019 SecureAuth Corporation
...
```

Similar in all other scripts.

### Additional context  
Space for additional context, investigative results, suspected issue.

**Comment by asolino** [comment]
We're planning on releasing next week. Keep in mind it is just a *plan*.

**Comment by noraj** [comment]
@asolino Nice. No worry no urgency :wink:

**Comment by noraj** [comment]
I'm waiting for 2 breaking fix https://github.com/SecureAuthCorp/impacket/pull/711 (digestmod) and https://github.com/SecureAuthCorp/impacket/commit/4a5403c3c15875a281a69f95f5fe02edcf6969e6 to fix https://github.com/fox-it/BloodHound.py/issues/41#issuecomment-602042628 that are already included in 0.9.21
